maps/aurora/aurora-5_interstitial.dmm

```diff
@@ -167,7 +167,9 @@
 /area/maintenance/research_port)
 "ar" = (
 /turf/simulated/wall/r_wall,
-/area/maintenance/arrivals)
+/area/engineering/workshop{
+	name = "\improper Construction Level"
```

Comment:
No, thats not a good idea.
If you need a area named construction level, then add it in the map defines for the aurora map.

Reply:
I'll see what I can do